### PR TITLE
new commit

### PR DESCRIPTION
**Pull request by Amangusain7**

### CODE REVIEW

File: Usermanagementsystem/userRegistration/models.py [hunk 1]
```diff
@@ -18,6 +18,7 @@ class User(AbstractBaseUser,PermissionsMixin):
     nationality = models.CharField(max_length=100,null=True,blank=True)
     gender = models.CharField(max_length=100,choices=GENDER_CHOICES,null=True,blank=True)
     date_joined = models.DateTimeField(default=timezone.now)
+    avatar = models.ImageField('profile picture', null=True, blank=True)
```
Comment (aayush-kumar-codes) [inline Usermanagementsystem/userRegistration/models.py]: change avatar to profile_image

File: Usermanagementsystem/userRegistration/views.py [hunk 2]
```diff
@@ -23,5 +23,30 @@ def post(self,request):
         else:
             return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    def put(self,request):
+        email = request.GET.get('email')
+        li = User.objects.get(email=email)
```
Comment (aayush-kumar-codes) [inline Usermanagementsystem/userRegistration/views.py]: variables name should be proper

File: Usermanagementsystem/userRegistration/authenticate.py [hunk 3]
```diff
@@ -0,0 +1,17 @@
+from userRegistration.models import User
```
Comment (aayushsaini000) [inline Usermanagementsystem/userRegistration/authenticate.py:1]: this one is wrong

File: Usermanagementsystem/userRegistration/views.py [hunk 4]
```diff
@@ -17,11 +19,41 @@ def post(self,request):
...
-                return Response({"token":token.key},status=status.HTTP_201_CREATED)
+                return Response({"token":token.key,"role":user.role},status=status.HTTP_201_CREATED)
             else:
-                return Response([],status=status.HTTP_400_BAD_REQUEST)
+                return Response("something went wrong",status=status.HTTP_400_BAD_REQUEST)
         else:
             return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
+    def put(self,request):
+        email = request.GET.get('email')
```
Comment (aayushsaini000) [inline Usermanagementsystem/userRegistration/views.py:30]: use authentication in update also

**Pull request merged**